package.json

```diff
@@ -41,7 +41,8 @@
     "react": "15.3.2",
     "react-dom": "15.3.2",
     "tinycolor2": "1.4.1",
-    "uuid": "2.0.3"
+    "uuid": "2.0.3",
+    "yargs-parser": "4.0.2"
```

Comment:
Heat it is needed. I tried to pack the app and then this was required. I added it and the build fixed. So I though somehow in the dependencies this one is forgotten, let's add this